### ISSUE TITLE
[WASM] `<Bold>` is not displayed properly on MacOS Chrome browser

### ISSUE THREAD

**Issue by TopProgrammer77** [unoplatform/uno#16514]
### Current behavior

The code which I tested is as follows.
`<TextBlock>To setup access to your <Bold>Apple iCloud account</Bold> you needed to create and assign an <Hyperlink NavigateUri="https://support.apple.com/en-us/HT204397">app-specific password</Hyperlink>.</TextBlock>`

Here, the text inside `<Bold>` is not displayed in MacOS Chrome browser. However, it works on Safari and Windows Chrome.

- On MacOS Safari (Good)
![image](https://github.com/unoplatform/uno/assets/122166904/74ee1523-d47c-430e-acbb-6029cadaec6e)

- On Windows Chrome (Good)
![image](https://github.com/unoplatform/uno/assets/122166904/0aec10e3-6da2-40f9-963a-dd223facf512)

- On Mac Chrome (Bad)
![image](https://github.com/unoplatform/uno/assets/122166904/cd395080-3e47-4a56-afaf-cfbf33f3ca55)

In other words, on Mac Chrome, `BOLD` is applied (maybe rendered to `font-weight: 700` CSS property) but the difference between bold text and normal text is not clear.

### Works on UWP/WinUI

None

**Comment by TopProgrammer77** [comment]
I think it is related to #14401

**Comment by agneszitte** [comment]
cc @jeromelaban, @MartinZikmund, @Youssef1313

**Comment by Youssef1313** [comment]
Let's re-open to keep tracking this

**Comment by TopProgrammer77** [comment]
The BOLD text is thin or regular text is thick? 😉

**Comment by DierkDroth** [comment]
@Youssef1313 if understand the info in my link above correctly then this clearly is an Apple issue.

Hence, I'm not sure what UNO could do there ... the only thing I could think of is applying my code snippet to the default WASM index.html?!?

**Comment by MartinZikmund** [comment]
@Youssef1313 Is this still happening after the switch to Open Sans?
